src/main/java/teammates/ui/controller/StudentFeedbackResultsPageData.java

```diff
@@ -177,7 +177,8 @@ private FeedbackResultsResponseTable createResponseTable(FeedbackQuestionAttribu
             boolean isUserPartOfGiverTeam = student.team.equals(giverName);
             if (question.giverType == FeedbackParticipantType.TEAMS
                 && isUserPartOfGiverTeam) {
-                giverName = "Your Team (" + giverName + ")";
+                giverName = new StringBuilder().append("Your Team (").append(giverName).append(")")
+                                               .toString();
```

Comment:
I see, perhaps someday I'll do a thorough review on all the suppressions and see which ones can be removed.